src/Pages/Overview/Overview.tsx

```diff
@@ -7,7 +7,7 @@ export const Overview = () => {
         <h1>Mission</h1>
         <p>
           To improve the software development lifecycle, no matter the size of
-          the project, by standardizing feature flagging for developers.
+          the project, by standardizing feature flagging for everyone.
```

Comment:
I'm not sure if this is right, or if we should simply use `Standardizing Feature Flagging for Everyone`

Reply:
That's our current mission statement. I think it's fine to keep it like you have it.

https://github.com/open-feature/community/blob/main/mission-vision.md